.gitallowed

```diff
@@ -0,0 +1 @@
+arn:[a-z0-9-:]+:[0-9]{12}:[a-z0-9-]+
```

Comment:
we tightened up the git allowed on another repositiory to just feature sns, do we want to do a similar thing here?

Reply:
Whilst I don't think we should be shouting ARNs from the rooftops, they're not something I think we should be considering a 'secret'. Down that road lies security through obscurity.

I'd rather work to move them all to an environment variable somewhere if we need to.